pages/docs/concepts/producer.md

```diff
@@ -0,0 +1,7 @@
+## What is a Producer? 
```

Comment:
<img width="1151" alt="Screen Shot 2022-06-13 at 3 50 37 PM" src="https://user-images.githubusercontent.com/19964402/173460299-baf9f52d-1b83-4ab4-b1d9-39c14d08601c.png">

just adding this for visual context so folks can see the first draft of your diagram and your understanding of this spec term so far .. this is good learning for me too! :)

Reply:
Thank @alequetzalli

Reply:
Hello @derberg , again thanks for your feedback. 

I need you to help me clarify some things:
-  You said, "Some producers send message to one topic, and some to the other one". I don't quite get that statement. 

Also, my understanding of a producer is that when an event occurs it sends that message to the broker, now what I want to know is does a producer have a particular channel in the broker where it sends its own messages so that any consumer subscribed to that channel can handle it?

Reply:
Yes it does help... Thank you @derberg